src/constants/common.ts

```diff
@@ -1,4 +1,5 @@
 export const ROUTES = {
   HOME: '/',
   BOOKINGS: '/bookings',
+  BOOKING_DETAILS: (id: string) => `/bookings/${id}`,
```

Comment:
No need to do that. React router resolves dynamic routes under the hood. Just use  `BOOKING_DETAILS: '/bookings/:id'`